src/k8s/pkg/k8sd/features/contour/chart.go

```diff
@@ -33,20 +33,29 @@ var (
 		Namespace:    "projectcontour",
 		ManifestPath: path.Join("charts", "ck-contour-common-1.28.2.tgz"),
 	}
+
+	// envoyProxyImageRepo represents the image to use for envoy in the gateway.
+	envoyProxyImageRepo = "ghcr.io/canonical/k8s-snap/envoyproxy/envoy"
+
+	// NOTE: The image version is v1.29.2 instead of 1.28.2
+	// to follow the upstream configuration for the contour gateway provisioner.
+	// envoyProxyImageTag is the tag to use for for envoy in the gateway.
+	envoyProxyImageTag = "v1.29.2"
+
 	// envoyImageRepo represents the image to use for the Contour Envoy proxy.
-	envoyImageRepo = "docker.io/bitnami/envoy"
+	envoyImageRepo = "ghcr.io/canonical/k8s-snap/bitnami/envoy"
 
 	// envoyImageTag is the tag to use for the Contour Envoy proxy image.
 	envoyImageTag = "1.28.2-debian-12-r0"
```

Comment:
It is quite easy to get lost with this, since these basically refer to the same image, without any distiction between them.

When updating the chart in the future, how do we know that we have not missed or mixed these images?

Use a specific name for each, in order to make it harder to miss, e.g. `contourGatewayProvisionerEnvoyImage....`, `contourIngressEnvoyImage...`